### ISSUE TITLE
[bug]: Minor issue iwth onyx expand mode texts

### ISSUE THREAD

**Issue by HaasJona** [freesewing/freesewing#6721]
### What seems to be the problem? 🤔

When I generate Onyx and toggle expand mode, the text for the crotch gusset is incorrect:

https://freesewing.org/new/onyx#view=%22draft%22&settings=%7B%22measurements%22%3A%7B%22biceps%22%3A310%2C%22neck%22%3A360%2C%22chest%22%3A870%2C%22waist%22%3A880%2C%22hips%22%3A880%2C%22seat%22%3A920%2C%22waistToHips%22%3A60%2C%22hpsToWaistBack%22%3A430%2C%22hpsToWaistFront%22%3A410%2C%22waistToSeat%22%3A230%2C%22crossSeam%22%3A690%2C%22waistToArmpit%22%3A190%2C%22inseam%22%3A840%2C%22upperLeg%22%3A520%2C%22waistToUpperLeg%22%3A250%2C%22ankle%22%3A230%2C%22wrist%22%3A165%2C%22shoulderToWrist%22%3A630%2C%22head%22%3A570%2C%22highBust%22%3A890%7D%2C%22units%22%3A%22metric%22%2C%22metadata%22%3A%7B%22setName%22%3A%22Ghislain%22%7D%2C%22options%22%3A%7B%22sleeveRibbing%22%3Atrue%7D%2C%22expand%22%3A0%7D

```
Cut one rectangle of main fabric
for the crotch gusset, with dimensions:
11.44cm x NaNcm
```

### Design / Plugin / Package 🧐

designs/onyx

**Comment by Thrunic** [comment]
I've created PR #6731 to fix this bug. Thank you for finding it.

The cause is that in the absolute options code for crotchGussetLength and crotchGussetWidth, I assumed that sa being off set it to zero. This is now fixed.

**Comment by Thrunic** [comment]
I've put in a PR (#6731) that should fix it. Thank you for finding the bug.

 
 

Sent: Friday, May 10, 2024 at 4:16 AM

**Comment by HaasJona** [comment]
@Thrunic are you sure that fixes the issue? I was referring to the text printed on the pattern parts, you commit only seems to affect the preview value of the sliders.

**Comment by Thrunic** [comment]
> @Thrunic are you sure that fixes the issue? I was referring to the text printed on the pattern parts, your commit only seems to affect the preview value of the sliders.
> 
> (I haven't actually tested your fix, so ignore this, if it somehow actually fixes the issue)

It should fix the issue. The text is based on the slider value, so a bug in the slider code is expected to result in a bug in the text.

**Comment by joostdecock** [comment]
I've checked and can confirm this is fixed in #6731. Closing.

@bobgeorgethe3rd You can only assign issues to users who are member of the FreeSewing organisation on GitHub.
@Thrunic was not a member, which is obviously an oversight, so I've sent them an invite and if/when they accept, you'll be able to assign issues to them.

If you come across other contributors who you cannot assign to, let me know because that almost certainly means that I forgot to add them.
